docs: add information for contributors (#3264)
process-analytics/bpmn-visualization-js: docs/contributors/development.md

Project, in major line, follows the code style suggested by [airbnb](https://github.com/airbnb/javascript) which is explicit and well documented.

> Typescript have been chosen as it's strongly typed, and we believe it adds some syntactical benefits to the JavaScript language

Reviewer: **suggestion:** 
```suggestion
Typescript was chosen because it's strongly typed, and we believe it adds some syntactical benefits to the JavaScript language.
```

## Typescript

> Although Linting, Sonar and Tests keeps the code in a good shape

Reviewer: **suggestion:** 
```suggestion
Although Linting, Sonar and Tests keeps the code in a good shape.

```

> We strongly recommend that you read the following resources to write code that conforms to best practices:

Reviewer: **suggestion:**  To be more English-friendly
```suggestion
To write code that conforms to best practices, we strongly recommend that you review the following resources:
```

> We strongly recommend you to read following resources to be able to write code that is conformed to the best practices:

Reviewer 2: ```suggestion
We strongly recommend that you read the following resources to write code that conforms to best practices:
```

Author: thanks
